src/wp-includes/block-supports/duotone.php

```diff
@@ -521,6 +554,26 @@ function wp_render_duotone_support( $block_content, $block ) {
 	wp_add_inline_style( $filter_id, $filter_style );
 	wp_enqueue_style( $filter_id );
 
+	add_action(
+		'wp_footer',
+		static function () use ( $filter_svg, $selector ) {
+			echo $filter_svg;
+
+			// Safari renders elements incorrectly on first paint when the SVG
+			// filter comes after the content that it is filtering, so we force
+			// a repaint with a WebKit hack which solves the issue.
```

Comment:
Core's coding standard for multi-line comments is this:

```suggestion
			/*
			 * Safari renders elements incorrectly on first paint when the SVG
			 * filter comes after the content that it is filtering, so we force
			 * a repaint with a WebKit hack which solves the issue.
			 */
```